sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2Suite.scala

```diff
@@ -367,31 +361,13 @@ class DataSourceV2Suite extends QueryTest with SharedSQLContext {
           val format = classOf[SimpleWritableDataSource].getName
 
           val df = Seq((1L, 2L)).toDF("i", "j")
-          df.write.format(format).option("path", optionPath).save()
+          df.write.format(format).option("path", optionPath).mode("append").save()
           assert(!new File(sessionPath).exists)
           checkAnswer(spark.read.format(format).option("path", optionPath).load(), df)
         }
       }
     }
   }
-
-  test("SPARK-25700: do not read schema when writing in other modes except append and overwrite") {
```

Comment:
This test is not needed anymore, as `ErrorIfExists` and `Ignore` modes are unsupported now.